<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CanteenWidget</class>
 <widget class="QWidget" name="CanteenWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="textEditRead">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>571</width>
     <height>491</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditWrite">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>441</width>
     <height>30</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请在此输入</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSend">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>510</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonClose">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>断开连接</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/ziyuan/bj.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>560</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>退出聊天</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonFile">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>560</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>选择菜谱</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>560</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>发送菜谱</string>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>textEditRead</zorder>
  <zorder>textEditWrite</zorder>
  <zorder>buttonSend</zorder>
  <zorder>buttonClose</zorder>
  <zorder>pushButton</zorder>
  <zorder>buttonFile</zorder>
  <zorder>pushButton_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
